invenio_rdm_records/requests/community_membership.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 CERN.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Community submission request."""
+
+from flask_babelex import lazy_gettext as _
+from invenio_requests.customizations import DefaultRequestType
+
+
+#
+# Request
+#
+class CommunityMembership(DefaultRequestType):
```

Comment:
Shouldn't this be in `invenio-communities`?

Reply:
Same as above